core-web/libs/portlets/dot-locales/portlet/src/lib/dot-locales-list/store/dot-locales-list.store.ts

```diff
@@ -16,64 +39,232 @@ export interface DotLocaleRow {
     actions: DotActionMenuItem[];
 }
 
-export interface DotLocalesListState {
+export interface DotLocalesListState extends DotLanguagesISO {
     status: ComponentStatus;
     locales: DotLocaleRow[];
+    initialLocales: DotLanguage[];
```

Comment:
What is the diff between `locales` and `initialLocales` and whay are diff types?

Reply:
good calll, actually wont need the `initialLocales` ( raw data from endpoint ), removed. `locales` is the data ready to be presented in the UI.